rasa/cli/x.py

```diff
@@ -111,11 +111,28 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
-    endpoints.model = EndpointConfig(
-        "{}/projects/default/models/tags/production".format(rasa_x_url),
-        token=rasa_x_token,
-        wait_time_between_pulls=2,
-    )
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs["wait_time_between_pulls"]
```

Comment:
Do a `kwargs.get`, otherwise this might raise an exception

Reply:
ok updated this for the `kwargs.get` instead now